packages/glib.rb

```diff
@@ -2,16 +2,17 @@
 
 class Glib < Package
   description 'GLib provides the core application building blocks for libraries and applications written in C.'
-  homepage 'https://developer.gnome.org/glib/'
-  version '2.40.2'
-  source_url 'https://ftp.gnome.org/pub/gnome/sources/glib/2.40/glib-2.40.2.tar.xz'
-  source_sha256 'e8ff8af2950897e805408480c454c415d1eade4e670ec5fb507f5e5853726c7a'
+  homepage 'https://developer.gnome.org/glib'
+  version '2.52.3'
+  source_url 'https://ftp.gnome.org/pub/gnome/sources/glib/2.52/glib-2.52.3.tar.xz'
+  source_sha256 '25ee7635a7c0fcd4ec91cbc3ae07c7f8f5ce621d8183511f414ded09e7e4e128'
 
   depends_on 'libffi'
   depends_on 'gettext'
+  depends_on 'util_linux'
```

Comment:
Oh, wait, you meant if this needs to be added?
In this case, no, `with-pcre` says to use glib-provided pcre.